server/schema/user/user.mutation.js

```diff
@@ -148,7 +147,10 @@ module.exports = new GraphQLObjectType({
     setUserRoles: {
       type: UserType,
       args: {
-        email: { type: new GraphQLNonNull(GraphQLString) },
+        id: {
+          description: "The user's mongo ID.",
+          type: new GraphQLNonNull(GraphQLID),
+        },
```

Comment:
Why use mongoid when we can directly user email to reduce steps? Both are unique anyway.